### PR TITLE
Form name in reverse order

### PR DESCRIPTION
**Pull request by Sheikh-Umar**

### CODE REVIEW

**Comment by davidten** [comment]
so from what I understand, your code will check if a name is reverse of another name and if it is it will say it's a duplicate?

File: src/main/java/seedu/address/logic/commands/AddCommand.java [hunk 1]
```diff
@@ -11,7 +11,7 @@
 import seedu.address.model.person.Person;
 import seedu.address.model.person.exceptions.DuplicatePersonException;
 
-/**
```
Comment (WoodyLau) [inline src/main/java/seedu/address/logic/commands/AddCommand.java]: Why are these asterisks deleted? Not that it is a bad thing, but maybe they were there for a reason?

Comment (Sheikh-Umar) [reply src/main/java/seedu/address/logic/commands/AddCommand.java]: Yes I realised that I had unnecessarily deleted it.

**Comment by Sheikh-Umar** [comment]
Yes. If the reverse of the name of a lead to be entered is the same as any of the names in the address book, it is a duplicate. However, this should be for leads with a first name and last name only, so a prerequisite for a lead is to have a first name and a last name.

**Comment by davidten** [comment]
I think this is too naive a way of finding duplicates, a step, yes but not enough to merge in

**Comment by WoodyLau** [comment]
I think a better way would be to match names like John Doe and Doe, John or Doe John, since we currently do not have first and last name fields.

**Comment by davidten** [comment]
Stale PR, closing.

**Pull request closed without merging**